src/test/integration/tests/eigenpod/FullySlashed_EigenPod.t.sol

```diff
@@ -1,22 +1,22 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.27;
 
-import "src/test/integration/IntegrationChecks.t.sol";
+import "src/test/integration/tests/eigenpod/EigenPod.t.sol";
 
-contract Integration_FullySlashedEigenpod_Base is IntegrationCheckUtils {
+contract Integration_EigenPod_FullySlashedEigenpod_Base is EigenPodTest {
     using ArrayLib for *;
 
-    User staker;
-    IStrategy[] strategies;
-    uint[] initTokenBalances;
-    uint[] initDepositShares;
-    uint64 slashedGwei;
-    uint40[] validators;
-
     function _init() internal virtual override {
+        super._init();
+
         _configAssetTypes(HOLDS_ETH);
         (staker, strategies, initTokenBalances) = _newRandomStaker();
-        cheats.assume(initTokenBalances[0] >= 64 ether);
+
+        // Ensure the staker has at least 64 ETH to deposit.
+        if (initTokenBalances[0] < 64 ether) {
+            initTokenBalances[0] = 64 ether;
+            cheats.deal(address(staker), 64 ether);
+        }
```

Comment:
we just need better "create user" syntax tbh. can leave that for a followup PR though.